### ISSUE TITLE
Feedback for Other vendors page

### ISSUE THREAD

**Issue by Dnominated** [urbandroid-team/dont-kill-my-app#175]
Ulefone:
This manufacturer's coders have added something called "DuraSpeed". 
If you disabled it*, all apps will be automatically restricted from running in the background. In other words, 'DuraSpeed' unbreaks a broken basic Android OS.

_*If you load a management app - any management app - the System menu will crash from the Settings menu. Such as using App Freezer (MobizSystems), a great app that works on my other phones without issue._

So you must leave DuraSpeed, and go into it and 'whitelist' every single app. 
That is -besides- doing the usual stock Android 'Don't optimize battery' on an app.

Tested on the Android 9 that came with the phone, 3/2020.

**Comment by Artaud** [comment]
DuraSpeed is a known villain from Nokia lower tier phones. So apparently this software is being bundled to other Chinese phones as well - we already knew that the Nokia/HMD Global phones come with the OS loaded with a bunch of crap because the company is a partner of FIH Mobile, which is a subsidiary of Foxconn. So apparently a lot of the battery killers in Chinese-designed phones come from a single source.

What you describe is quite alarming but big thanks for sharing the tip!

Could you please also tell me the exact model?

**Comment by Dnominated** [comment]
3wt. But you are going to find this with any of their current models.

Key thing is the android itself is broke, because if you use a management app to disable DuraSpeed, it seems that automatically places every app on a default 'blacklist', and they are killed some set amount of time after the screen goes off.

**Comment by Artaud** [comment]
But Doze mode in base Android doesn't terminate apps - it just defers tasks like jobs and alarms and forces you to use foreground services instead of background services, regardless of the optimized/non-optimized mode.
What you are talking about is definitely a vendor modification of Ulefone/Gotron, not the behaviour of AOSP or Google-flavored Android.

**Comment by Dnominated** [comment]
Yes. I am referring to the chinesium breaking/broken version of android. Not AOSP.
